### PR TITLE
Update pe.m3u

### PR DESCRIPTION
**Pull request by sguinetti**
This PR adds new peruvian channels

### CODE REVIEW

File: channels/pe.m3u [hunk 1]
```diff
@@ -51,3 +51,15 @@ https://cdnh8.iblups.com/hls/RMuwrdk7M9.m3u8
...
+#EXTINF:-1 tvg-id="" tvg-name="Willax" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/willaxtv/picture?width=320&height=320" group-title="",Willax
+https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.dailymotion.com/WillaxTV
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/antaresTvRadioOficial/picture?width=320&height=320" group-title="Local",Antares Televisión
+https://5c3fb01839654.streamlock.net:554/iptvantares/liveantarestv/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/CongresoPeru/picture?width=320&height=320" group-title="",Congreso de la República (Peru)
+https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.youtube.com/channel/UCsKiP5cZCYh9YhPGrI6GrkQ
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/CadenaTVHuancayo/picture?width=320&height=320" group-title="Local",Cadena TV Huancayo
+https://edge1o.live.opencaster.com/caster/kAaEdjbBiibk.m3u8
```
Comment (Dum4G) [inline channels/pe.m3u:61]: Can't open that link

Comment (sguinetti) [reply channels/pe.m3u:61]: Ok, I will check the link

Comment (themak1985) [reply channels/pe.m3u:61]: I couldn't get either of them to work.

Comment (sguinetti) [reply channels/pe.m3u:61]: The website is https://www.cadenatv.com.pe/en-vivo/, but the broadcasting works in this web. Any suggestions?

Comment (Dum4G) [reply channels/pe.m3u:61]: Casual problem: they want proper html refferer
Referrer Policy: strict-origin-when-cross-origin

Comment (Dum4G) [reply channels/pe.m3u:61]: @LaneSh4d0w please maintain your hosting and make sure it resets automatically

Comment (sguinetti) [reply channels/pe.m3u:61]: Like this?:
#EXTVLCOPT:http-referrer=https://www.opencaster.com
#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/CadenaTVHuancayo/picture?width=320&height=320" group-title="Local",Cadena TV Huancayo
https://edge1o.live.opencaster.com/caster/kAaEdjbBiibk.m3u8

Comment (LaneSh4d0w) [reply channels/pe.m3u:62]: > @LaneSh4d0w please maintain your hosting and make sure it resets automatically
> https://github.com/iptv-org/iptv/pull/2933#discussion_r641739353

Thanks for noticing. It restarted.

Comment (Dum4G) [reply channels/pe.m3u:62]: #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/CadenaTVHuancayo/picture?width=320&height=320" group-title="Local",Cadena TV Huancayo
#EXTVLCOPT:http-referrer=https://www.opencaster.com
https://edge1o.live.opencaster.com/caster/kAaEdjbBiibk.m3u8

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/pe.m3u (hunk 1 above)
Comment (sguinetti) [reply channels/pe.m3u:62]: > #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/CadenaTVHuancayo/picture?width=320&height=320" group-title="Local",Cadena TV Huancayo
> #EXTVLCOPT:http-referrer=https://www.opencaster.com
> https://edge1o.live.opencaster.com/caster/kAaEdjbBiibk.m3u8

Done

**Comment by Dum4G** [comment]
Didn't help. I think I'm missing something

**Comment by sguinetti** [comment]
I will remove the problematic link to fix it in the future.

**Review by freearhey — APPROVED** [review]

**Comment by LaneSh4d0w** [comment]
> 
> 
> Didn't help. I think I'm missing something

It's referer locked, simply. Found out by modifying the headers of the request.

**Comment by Dum4G** [comment]
> It's referer locked, simply. Found out by modifying the headers of the request.

We tried to specify the referrer Chrome mentions as we usually do in VLC but it didn't help

**Comment by LaneSh4d0w** [comment]
image

Needed to add Origin and Referer.

**Pull request merged**